doc/development_setup.md

```diff
@@ -53,6 +53,24 @@ If you do more development on transcribee, you may wish to do the following thin
 - Install the [`pre-commit`](https://pre-commit.com/) hook so that your changes are automatically
   linted before you commit them. Run: `pre-commit install`
 
+## Add pages
+
+`transcribee` contains a minimal page system.
+To add pages, add a `pages.json` in `backend/data`.
+You can generate such a `pages.json` from a directory of markdown-files using the following command in the `backend/`-folder.
+
+```shell
+pdm run generate_pages_json PATH/TO/MARKDOWN/FOLDER data/pages.json
+```
+
+You can select which pages should be always shown in the footer using the `--footer-pages` option to the script. For example if you wanted the content of `page1.md` and `page2.md` to be shown in the footer, run
+
+```
+pdm run generate_pages_json PATH/TO/MARKDOWN/FOLDER data/pages.json --footer-pages page1 page2
+```
+
+The link text is automatically generated from the first first-level heading in the document.
+
```

Comment:
What do you think about yaml front matter and no pages.json? The application could just scan the pages folder on startup and generate the rest.

Reply:
I thought about that as well and was undecided. Probably the easier solution